gr-utils/modtool/complete_zsh.in

```diff
@@ -0,0 +1,41 @@
+#compdef gr_modtool
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Hey, don't forget to include your name there, too :)

Reply:
The files were generated with `_GR_MODTOOL_COMPLETE=zsh_source gr_modtool`, etc. That notice should probably include Pallets, because it's [their code](https://github.com/pallets/click/blob/e16088a8569597c55f108ea89af6245898249ec2/src/click/shell_completion.py#L127).

If I add more features (e.g. block names), I will include my name. ;)